adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -1011,6 +1355,19 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
     }
   }
 
+  /**
+   * Loads alignments from a given path, and infers the input type.
```

Comment:
should this be alignment fragments?  Or better yet:  Auto-detects the file type and loads a FragmentRDD.